
{'date': 'Tue Aug 06 2019 19:20:19.25 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 18
  class doidera()
                  ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Aug 06 2019 19:20:38.960 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 29
    doidera()
  module <module> line 22
    dr_zukman-Elemento(img=DR_ZUKMAN)
NameError: name 'dr_zukman' is not defined
'''},
{'date': 'Tue Aug 06 2019 19:20:55.27 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 29
    doidera()
  module <module> line 26
    dr_zukmam.entra(congresso.norte)
NameError: name 'dr_zukmam' is not defined
'''},
{'date': 'Tue Oct 01 2019 18:48:29.231 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 2
  from _spy.vitollino.main import Cena, Texto,  """Elemento """, STYLE,Sala
                                                 ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Oct 01 2019 18:52:51.227 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 25
  class Personagens():
  ^
IndentationError: expected an indented block
'''},
{'date': 'Tue Oct 01 2019 18:53:57.903 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 44
  def entra_porta(self,evento,nome):
  ^
IndentationError: unexpected indent
'''},
{'date': 'Tue Oct 01 2019 18:58:15.74 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 50
    Cenas()
  module <module> line 43
    ana_maria.entra(congresso.norte)
NameError: name 'ana_maria' is not defined
'''},
{'date': 'Tue Oct 01 2019 19:00:15.761 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 52
    Cenas()
  module <module> line 46
    dr_zukman.entra(congresso.norte)
NameError: name 'dr_zukman' is not defined
'''},
{'date': 'Tue Oct 01 2019 19:00:53.616 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 52
    Cenas()
  module <module> line 47
    porta.entra(congresso.norte)
AttributeError: 'porta' object has no attribute 'entra'
'''},
{'date': 'Tue Oct 01 2019 19:03:51.0 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 54
    Cenas()
  module <module> line 49
    Porta().entra(congresso.norte)
  module <module> line 38
    dragger=dict(Aninha=self.entra_porta)
AttributeError: 'Porta' object has no attribute 'entra_porta'
'''},